src/lib/eccentric.css

```diff
@@ -126,4 +126,18 @@ h5, .et-h5 {
 h6, .et-h6 {
   font-size: 12.94px;
 }
+
+.et-paragragh, .et-bold, .et-italics, .et-underlined, .et-quote, .et-regular, p, b, i, u, q {
+  font-family: var(--et-sans-serif);
+  margin: 0;
+  color: var(--et-black);
+}
+
+.et-bold, b {
+  font-weight: bold;
+}
+
+.et-paragragh, .et-regular, p {
```

Comment:
There's no need to specify CSS at this block as the properties are already defined for this.
Also, We need to create a method named as `.et-semibold` and set font-weight as 600.

But that's fine, LGTM!

I will apply the remaining changes.